app/views/core_induction_programme/lesson_parts/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= govuk_breadcrumbs breadcrumbs: course_lesson_breadcrumbs(@current_user, @course_lesson_part.course_lesson) %>
+<div class="govuk-grid-row">
```

Comment:
Just a suggestion - adding couple of extra empty lines will speed up reading the file - cognitively we store information in chunks and then form relationships between those chunks